src/main/proto/sqlflow.proto

```diff
@@ -63,12 +63,16 @@ message FetchRequest {
 }
 
 message FetchResponse {
-    message Logs {
-        repeated string content = 1;
+    message Logs { //  depreated
+                   repeated string content = 1;
+    }
+    message Responses {
+        repeated Response response = 1;
     }
     FetchRequest updated_fetch_since = 1;
     bool eof = 2;
-    Logs logs = 3;
```

Comment:
Maybe we also need to update the code of fetching logs, in the newly protobuf, the log would be in `Response.Message`